doc/policies/authentication.rst

```diff
@@ -587,3 +587,17 @@ supported by the token.
 
 .. note:: If you configure this, you will likely also want to configure
     :ref:`policy_webauthn_enroll_user_verification_requirement`.
+
+
+question_numbers
+~~~~~~~~~~~~~~~~
+
+type: integer
+
+The questionnaire token can ask more than one question during one authentication process.
+It will ask the first question, verify the answer, ask the next question and verify the answer.
+This policy setting defines how many questions the user needs to answer. (default: 1)
+
+
+.. note:: It is currently not verified, that the same question is asked more than once, so it is a good
+   idea to have a lot more possible answers than questions being asked.
```

Comment:
OK, we could probably store the *ID* of the tokeninfo entry of the question in the *Challenge* table.

Reply:
Done in 6dad6bc7